### PR TITLE
Feature cnm survey design

### LINKED ISSUE #1867: Change CNM Survey design

### PR DESCRIPTION
**Pull request by manuelplazaspalacio**
Close #1867

### CODE REVIEW

**Comment by manuelplazaspalacio** [comment]
This issue needs issue #1862 to be merged before.

**Review by ifoche — CHANGES_REQUESTED** [review]
@manuelplazaspalacio a few comments that come probably as the result of the merge with so many modifications in csvs. I'm sure you can solve this much more quickly than me. 

- In Diagnosis method question, options should be centered
screenshot_20180313-073049

- In test result, could the images be a bit bigger? I guess it's a matter of just adjusting a bit more the image to the edges of the file
screenshot_20180313-073116

- When 'Not tested' option is clicked, it takes you to stock question
screenshot_20180313-073149

Thanks!

File: app/src/cnm/assets/Questions.csv [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+11;'Code: Diagnosis Type';'ipc_issueEntry_q_diagnosis_type';;'ipc_issueEntry_q_diagnosis_type';'o7mG7zUBGsM';10;0;0;3;11;;32;7;0;;1
+12;'Code: Treatment';'ipc_issueEntry_q_treatment';;'ipc_issueEntry_q_treatment';'rBonfqEGNdS';11;0;0;3;12;;14;7;0;;1
+13;'Code: Other Treatment';'ipc_issueEntry_q_treatment_other';;'ipc_issueEntry_q_treatment_other';'V8q06yRjHqg';12;0;0;3;13;;3;7;0;;1
+14;'MCS STK - ASMQ 100/200 m';'ipc_issueEntry_q_stock_asmq_100_200';;'ipc_issueEntry_q_stock_asmq_100_200';'Y0rvgxJm28P';1;0;0;4;15;;26;4;0;;1
+15;'MCS STK - ASMQ 25/50 mg';'ipc_issueEntry_q_stock_asmq_25_50';;'ipc_issueEntry_q_stock_asmq_25_50';'WA1ouIgQyCr';2;0;0;4;16;;26;4;0;;1
+16;'MCS STK – Primaquine';'ipc_issueEntry_q_stock_primaquine';;'ipc_issueEntry_q_stock_primaquine';'LLP04MYE6LM';3;0;0;4;17;;26;4;0;;1
+17;'MCS STK – RDTs';'ipc_issueEntry_q_stock_rdt';;'ipc_issueEntry_q_stock_rdt';'wYekTzcJE6c';4;0;0;4;18;;26;4;0;;1
+18;'Code: Not Tested Reason';'ipc_issueEntry_q_not_reason_tested';;'ipc_issueEntry_q_not_reason_tested';'h34mDGMtzuW';2;0;0;1;19;;32;2;0;;1
+19;'Code: Referred';'ipc_issueEntry_q_referred';;'ipc_issueEntry_q_referred';'Weypkpeu9Z8';3;0;0;1;20;;32;8;0;;1
```
Comment (ifoche) [inline app/src/cnm/assets/Questions.csv]: @manuelplazaspalacio there's a conflict with this 2 DEs that has Weypkpeu9Z8 UID. Actually there's no DE in the server with that UID. Can you have a look? Thanks!

**Review by ifoche — CHANGES_REQUESTED** [review]

**Review by ifoche — APPROVED** [review]
thanks @manuelplazaspalacio

**Pull request merged**